packages/web/lib/browser.js

```diff
@@ -112,23 +108,24 @@ const getBrowserScope = (parent) => ({
       });
       return null;
     },
-    page: (scope) => (_opts) => (pattern, ...ruleSets) => {
-      assertBrowserScope(scope, "Cannot call page outside of a Browser scope");
+    // Override print to stderr, so that stdout output is reserved for the
+    // data extraction and can be piped easily
+    print: (_) => (_) => (...args) => {
+      console.error(...args.map(stringify));
+      return null;
+    },
+    page: (_) => ({ scrape = true }) => (pattern, ...ruleSets) => {
       const urlObj = url.parse(pattern);
 
       if (!urlObj || !urlObj.host) {
-        throw new BrowseError({
-          message: `'${pattern}' is not a valid URL pattern`,
-          node: null,
-        });
+        throw new Error(`'${pattern}' is not a valid URL pattern`);
```

Comment:
These are pretty confusing for me. I have trouble figuring out when to use Error vs BrowseError. Would you mind writing a quick comment about it in error.js for future reference.

Reply:
Okay, adding it. But here it is if you wanna read. tl;dr, they're the same thing

```
/**
 * A BrowseError which tracks the browse callstack. This is primarily for Rule
 * and RuleSet evaluations to use since we don't have lines we need to add to
 * the stacktrace anywhere else
 *
 * throwing a BrowseError that has no `node` set is the same as throwing a
 * regular Error, since the nearest evalRule or evalRuleSet will catch it and
 * use BrowseError.from to wrap it in a BrowseError
 */
```